modules/apps/staging/staging-test/src/testIntegration/java/com/liferay/staging/test/StagingImplTest.java

```diff
@@ -268,6 +272,36 @@ public void testRemoteStaging() throws Exception {
 		enableRemoteStaging(false);
 	}
 
+	@Test
+	public void testRemoteStagingHiddenError() throws Exception {
+		Throwable caughtThrowable = _enableRemoteStagingWithError();
```

Comment:
@vendeltoreki Please also set the property explicitly to make sure this integration test not depending on your current setting.